### ISSUE TITLE
New ZoneStatus Codes 32, 96

### ISSUE THREAD

**Issue by gibwar** [craigjmidwinter/total-connect-client#191]
Hi there! Running HA 2022.10.1 and `total-connect-client` 2022.5 and over the weekend (on vacation, no less) ended up getting a trouble fault with a low battery and bypass.

```
Logger: total_connect_client.zone
Source: components/totalconnect/__init__.py:100
First occurred: October 9, 2022 at 19:38:00 (1290 occurrences)
Last logged: October 10, 2022 at 06:23:46

unknown ZoneStatus 32 in OrderedDict([('AlarmTriggerTime', None), ('AlarmTriggerTimeLocalized', None), ('ZoneTypeID', 1), ('DeviceType', 0), ('ZoneID', 13), ('ZoneDescription', 'Shed Door'), ('ZoneStatus', 32), ('PartitionID', 1), ('CanBeBypassed', 1)])
unknown ZoneStatus 96 in OrderedDict([('AlarmTriggerTime', None), ('AlarmTriggerTimeLocalized', None), ('ZoneTypeID', 1), ('DeviceType', 0), ('ZoneID', 13), ('ZoneDescription', 'Shed Door'), ('ZoneStatus', 96), ('PartitionID', 1), ('CanBeBypassed', 1)])
```

The 32 is the first code that matches with the history entry "Sensor 13: Shed Door Sensor Communication Failure" and about 1:38 AM and changes to 96 which matches up with "Sensor 13: Shed Door Sensor Low Battery" and that goes on until I disarm and bypass the sensor and rearm.

Let me know what info you need from me for this request! (Additionally, if we can get bypass information in to HA that could help out my fan automations a lot when having the upstairs bedroom windows open at night during the summer, totally get that it needs to be a separate feature request)

**Comment by austinmroczek** [comment]
Please share your alarm panel model and sensor device model

**Comment by gibwar** [comment]
The sensor is one of the newer ProSix sensors, specifically the Mini Door/Window Sensor: PROSIXMINI2. The panel is a PROA7PLUS. Is there any more info you need?

**Comment by austinmroczek** [comment]
Should be fixed in the latest HA 2022.10.4.  

Guessing you already fixed the issue with your device and can't recreate the issue, but if not please let me know if it acts properly now.
